CoffeeChat/Cells/OnboardingTableViewCell.swift

```diff
@@ -9,7 +9,6 @@
 import UIKit
 
 class OnboardingTableViewCell: UITableViewCell {
-
```

Comment:
do you know what the convention is for new line or not after class declaration in the other files? nitpicky, but it's nice to keep it consistent